### DISCUSSION TITLE
请恢复划词翻译与设置其快捷键的功能。

### DISCUSSION THREAD

**Discussion by weshion** [theowenyoung/Traduzir-paginas-web#9] [Q&A]
最新版本中没有划词翻译功能了。可是这个功能是经常被使用的，有时候翻译整个页面出现问题时，经常需要用到划词翻译。请恢复划词翻译与设置其快捷键的功能。

**Comment by theowenyoung** [comment]
你好，由于精力关系， 我决定让这个插件只聚焦于内容类网页的翻译（双语对照）功能，对于划词翻译，市面上有非常多优秀的替代品，比如我在用的沙拉查词： https://saladict.crimx.com/ ，非常好用，比我这个插件原先附带的划词翻译更完善。也许可以考虑？
